doc/release-notes-1.10.rst

```diff
@@ -171,5 +171,353 @@ Issue Related Items
 
 .. comment  List derived from Jira/GitHub Issue query: ...
 
-* :jira:`ZEP-248` - reference a (public) Jira issue
-* :github:`1234` - reference a GitHub issue
+* :github:`1280` - shell on Arduino Due prints "shell>" before the delayed boot banner
```

Comment:
How about sorting by issue number?

Reply:
I can, they are currently sorted by closed date (not that anyone knows that).

Reply:
I was also surprised that Sphinx didn't complain about underscores.  Things like ``spi_legacy.h`` look like a hyperlink reference to something named ``spi``, so we've had to escape that by writing ``spi\_legacy.h``.  But didn't see the CI build failing, so guess the current version of Sphinx is more clever than previous versions.